[2025-03-15T12:29:53.463+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T12:29:53.478+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T12:29:51.754607+00:00 [queued]>
[2025-03-15T12:29:53.486+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T12:29:51.754607+00:00 [queued]>
[2025-03-15T12:29:53.488+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-03-15T12:29:53.506+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-15 12:29:51.754607+00:00
[2025-03-15T12:29:53.511+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=265) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T12:29:53.513+0000] {standard_task_runner.py:72} INFO - Started process 267 to run task
[2025-03-15T12:29:53.513+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'manual__2025-03-15T12:29:51.754607+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmpn9u9vcxk']
[2025-03-15T12:29:53.516+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask api
[2025-03-15T12:29:53.563+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api manual__2025-03-15T12:29:51.754607+00:00 [running]> on host 5f0b9577db8d
[2025-03-15T12:29:53.640+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T12:29:51.754607+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T12:29:51.754607+00:00'
[2025-03-15T12:29:53.641+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-15T12:29:54.015+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-15T12:29:54.039+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-15T12:29:54.040+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=manual__2025-03-15T12:29:51.754607+00:00, execution_date=20250315T122951, start_date=20250315T122953, end_date=20250315T122954
[2025-03-15T12:29:54.092+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T12:29:54.109+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T12:29:54.112+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
